### PR TITLE
For Issue #93 - Renamed cap and successMinimum to hardCap and softCap respectively and corrected calculation for seedToken amount.

### LINKED ISSUE #89: Bug: deploySeed sends “cap” amount of seed tokens to the contract, but the cap is denominated in funding tokens
On line 101, the `cap` for the `Seed` is set, and on line 113, seed tokens are sent to the `Seed` contract to the amount of cap. However, in  the seed contract, the cap is denoted in funding tokens, so, depending on the price, either too much or too little seed tokens are sent. This can lead to a loss of user’s funds, as they may get locked in the contract when the `Seed` contract runs out of seed tokens before the user has claimed hers. 
`TODO`:
- calculate the right amount of seed tokens to send

### LINKED ISSUE #93: rename cap and successMinimum
`cap` and `successMinimum` success minimum variables are named incorrectly and should be renamed to`hardCap` and `softCap`

### PR DESCRIPTION
**Pull request by codebuster22**
Resolves #93 
Resolves #89 

Renamed cap and successMinimum to hardCap and softCap respectively
Corrected calculations for ```seedToken``` amount at ```SeedFactory.sol```.

### CODE REVIEW

**Comment by codebuster22** [comment]
Do Not Merge, having some issue with the test, Working on it. Wanted to test on different enviornment.
The error I'm facing like this. 
```"
Contract: Staking: 1 week happypath
       "before all" hook: prepare suite in "Contract: Staking: 1 week happypath":
     ProviderError: 
Could not connect to your Ethereum client with the following parameters:
    - host       > 127.0.0.1
    - port       > 9545
    - network_id > 5777
Please check that your Ethereum client:
    - is running
    - is accepting RPC connections (i.e., "--rpc" option is used in geth)
    - is accessible over the network
    - is properly configured in your Truffle configuration file (truffle-config.js)```

**Comment by codebuster22** [comment]
That's weird, some test fails when running tests using ```truffle test```. But when the tests are run individually on a single file, they all pass. But the tests pass on the CI.

File: contracts/seed/Seed.sol [hunk 1]
```diff
@@ -137,8 +137,8 @@ contract Seed {
     ) public initializer {
         beneficiary     = _beneficiary;
         admin           = _admin;
-        successMinimum  = _successMinimumAndCap[0];
-        cap             = _successMinimumAndCap[1];
+        softCap         = _successMinimumAndCap[0];
```
Comment (kinspotting) [inline contracts/seed/Seed.sol:140]: pls adjust `_successMinimumAndCap`  to the updated naming, could `softAndHardCap` or something similar

File: contracts/seed/Seed.sol [hunk 2]
```diff
@@ -167,12 +167,12 @@ contract Seed {
         uint lockedSeedFee = (alreadyLockedSeedTokens.mul(uint(PPM))).mul(fee).div(PPM100);
 
         // We are that overall supply of fundingToken that will be in the end of this function execution
```
Comment (kinspotting) [inline contracts/seed/Seed.sol:169]: pls fix or delete the comment, it's not clear what it is saying

**Review by kinspotting — COMMENTED** [review]
needs minor changes

**Review by kinspotting — CHANGES_REQUESTED** [review]
needs minor changes

File: contracts/seed/Seed.sol (hunk 1 above)
Comment (kinspotting) [reply contracts/seed/Seed.sol:140]: pls do it in the `SeedFactory` contract as well

File: contracts/seed/Seed.sol (hunk 2 above)
Comment (codebuster22) [reply contracts/seed/Seed.sol:169]: Fixed this comment with PR #117

File: contracts/seed/Seed.sol (hunk 1 above)
Comment (codebuster22) [reply contracts/seed/Seed.sol]: Updated it.

**Comment by codebuster22** [comment]
Don't Merge, working on issue #89, as the ```cap``` is now renamed to ```hardCap``` and need ```cap``` for calculation of ```seedAmount``` to send.

**Review by kinspotting — APPROVED** [review]
lgtm

**Pull request merged**